sources/core/objects/Human.cpp

```diff
@@ -509,7 +529,7 @@ void Human::Bump(IdPtr<IMovable> item)
 
         unsigned int blood_value = (GetRand() % 7) + 1;
 
-        if (IdPtr<Floor> floor = GetTurf())
+        if (IdPtr<Floor> floor = GetTurf()) // TODO Bleeding only with ballistic bullets
```

Comment:
I could make it to trigger only after certain amount of damage( like a punch wouldn't trigger this effect) and at high brute damage there could be constant bleeding (per tick)(after 60% of hp lose in brute).